tools/lsp/preview/drop_location.rs

```diff
@@ -4,12 +4,12 @@
 use i_slint_core::lengths::{LogicalLength, LogicalPoint};
 use slint_interpreter::ComponentInstance;
 
-use crate::preview::element_selection::collect_all_element_nodes_covering;
-
 #[cfg(target_arch = "wasm32")]
 use crate::wasm_prelude::*;
 
-use super::element_selection::ElementRcNode;
+use super::element_selection::{
```

Comment:
I've been more or less following the guideline of importing types, but not functions. I remember reading that guideline somewhere but I can't find it anymore.